django_filters/filters.py

```diff
@@ -235,6 +240,12 @@ def filter(self, qs, value):
 
         return qs.distinct() if self.distinct else qs
 
+    def get_filter_predicate(self, v):
+        try:
+            return {self.name: getattr(v, self.field.to_field_name)}
+        except AttributeError:
+            return {self.name: v}
```

Comment:
Almost nobody ever uses it as far as I can tell (but it's very handy when needed). 

I can't see a problem.